docs/alerts.md

```diff
@@ -20,10 +20,82 @@
 ## Elasticsearch Cluster Health is Red
 
 At least one primary shard and its replicas are not allocated to a node.
+The most common causes can be failed cluster nodes and the elasticsearch process crashing due to a continuous heavy processing load.
```

Comment:
I don't know that we should state this is the most common case because we can also see the cluster is `red` when it is initially recovering from a restart (either a node or the full cluster)

Reply:
Got it, I have removed.